### ISSUE TITLE
The Myriad was resetting when using my own model

### ISSUE THREAD

**Issue by zhoujustin** [google/aiyprojects-raspbian#361]
Hi!
I'm try to deployed my model in VisionKit!
I'm scratch a new model with 160res_0.5 as the previous issues mentioned.
When I deployed the model in VisionKit, the dmesg out that:
[ 7565.530179] aiy-vision spi0.0: Resetting myriad
[ 7567.715881] aiy-vision spi0.0: Writing myriad firmware
[ 7575.416481] aiy-vision spi0.0: Myriad booting
[ 7575.597658] aiy-vision spi0.0: Myriad ready
[ 8028.335782] aiy-vision spi0.0: Slave not ready after 1000 ms
[ 8028.335805] aiy-vision spi0.0: Fatal error, resetting
and when I debug logging, it's output that:
INFO:root:InferenceEngine transport: _SpiTransport
INFO:root:Loading model "object_detection"...
Inference
INFO:root:Image inference with model "object_detection"...
INFO:root:Unloading model "object_detection"...
Traceback (most recent call last):
  File "/opt/aiy/projects-python/src/aiy/_drivers/_spicomm.py", line 137, in transact
    fcntl.ioctl(self._dev, SPICOMM_IOCTL_TRANSACT, buf)
OSError: [Errno 14] Bad address

The firmware version is 
cat /sys/bus/i2c/devices/1-0051/status_message
OK-V0.5
cat /sys/firmware/devicetree/base/hat/product_ver
0x0002

By the way, the demo of the 'out of box' was worked well.
Thanks!

**Comment by dmitriykovalev** [comment]
Could you please put an explicit link to the previous issues, so everything is connected.

**Comment by weiran-work** [comment]
@zhoujustin Is this a mobilenet + ssd model? If so, the only config that's supported right now is 
256x256 depthwise 0.125. And you will need this config
https://github.com/tensorflow/models/blob/master/research/object_detection/samples/configs/embedded_ssd_mobilenet_v1_coco.config

**Comment by zhoujustin** [comment]
@dmitriykovalev About this issue https://github.com/google/aiyprojects-raspbian/issues/314
@weiranzhao Thanks, I'm try to retrain the model from 256/0.125. export the ckpt by exprot_inference_graph and bonnet_model_compiler.par the graph. When deployed the graph into the VisionKit, I got the same result.
My ckpt, pb and binaryproto can be found:
https://drive.google.com/file/d/1hocJp3C3XzinyvYwqdTnIitJMc6aLN7B/view?usp=sharing

**Comment by zhoujustin** [comment]
@weiranzhao Seems like I found the problems of it. The trainning should be "embedded ssd". There's another question, how could I generate the "*_anchors.py" file?
Thanks.

**Comment by weiran-work** [comment]
@zhoujustin were you able to run your model on device using embedded ssd?
as for anchors.py, you can try to have a function similar to this
https://github.com/google/aiyprojects-raspbian/blob/aiyprojects/src/examples/vision/mobilenet_based_classifier.py#L26

It reads from a file with labels. label file just list all the labels with correct order, e.g.,
...
dog,
cat,
person,
grass,
...

**Comment by zhoujustin** [comment]
@weiranzhao 
Is that the --input_layer Preprocessor/sub and --output_layer concat,concat_1?
I print the result and find the concat, concat_1's data? How can I use it?

**Comment by zhoujustin** [comment]
@weiranzhao 
I confused that the binaryproto was created by bonnet_model_compiler. When I deployed it to the vision kit, I got the "len(logit_scores): 10224" assert.
I'm dump both my pb and the person_dog_cat's pb, There are some different between of that. 
There is my model https://drive.google.com/file/d/1bvc6IPwiS-nERJUOErOXyYFr5mHDPgDE/view?usp=sharing
The my.pbtxt is export from frozen pb, and the person_cat_dog.pbtxt is export from person_cat_dog.pb. 
Thanks!
